### ISSUE TITLE
TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by shunkakinoki** [NotionX/react-notion-x#17]
Thanks for the awesome project!

I get an error whenever I'm trying to load a database that is a page

```
TypeError: Cannot read property 'length' of undefined
    at Object.exports.getPageTableOfContents (/Users/shunkakinoki/ghq/github.com/shunkakinoki/shunkakinoki.com/node_modules/notion-utils/build/cjs/get-page-table-of-contents.js:42:46)
    at exports.Block (/Users/shunkakinoki/ghq/github.com/shunkakinoki/shunkakinoki.com/node_modules/react-notion-x/build/cjs/block.js:75:39)
    at processChild (/Users/shunkakinoki/ghq/github.com/shunkakinoki/shunkakinoki.com/node_modules/react-dom/cjs/react-dom-server.node.development.js:3373:14)
```

**Comment by transitive-bullshit** [comment]
Hey @shunkakinoki would you mind including a public notion page that I can repro the issue at? It'll just make it a lot quicker for me to debug what's going on. Thanks!

**Comment by transitive-bullshit** [comment]
This should now be fixed as of `3.4.8`.

Sorry about that.

**Comment by shunkakinoki** [comment]
Sorry for the late reply!
It's fixed now
Super appreciate your work!
